Mathlib/SetTheory/Game/Short.lean

```diff
@@ -205,9 +205,7 @@ instance listShortGet :
     ∀ (L : List PGame.{u}) [ListShort L] (i : Fin (List.length L)), Short (List.get L i)
   | [], _, n => by
     exfalso
-    rcases n with ⟨_, ⟨⟩⟩
-    -- Porting note: The proof errors unless `done` or a `;` is added after `rcases`
-    done
+    rcases n with ⟨_, ⟨⟩⟩;
```

Comment:
Seems like you haven't addressed the porting note, and removing `done` isn't a particularly great idea either.

I suggest you nolint `done` calls by default.

Reply:
Isn't the issue that `rcases` does not realize that its pattern is over, since `|` could be an `or`?  Again, I can put the porting note back, but replacing `done` by `;` also "fixes" the error.

Reply:
Here is an example:
```lean
example (x : Bool) (h : false ∨ true) : True := by
  match x with
    | true => rcases h with ⟨⟩
    | false  -- <--- this is still part of `rcases`: I am calling `false` the `true` proof! x)
              exact .intro  -- <--- the proof if `h` is `false`
              exact .intro  -- <--- the proof if `h` is `true`
    | false => exact .intro -- <--- the second `match` arm!

```